.github/dependabot.yml

```diff
@@ -0,0 +1,10 @@
+version: 2
+enable-beta-ecosystems: true
+updates:
+    - package-ecosystem: npm
```

Comment:
We use pnpm, and unfortunately dependabot doesn't support pnpm.

Reply:
See:
https://github.com/dependabot/dependabot-core/pull/7081
https://github.com/dependabot/dependabot-core/pull/7281

Reply:
It seems it is only available through beta testing and the comments you shared mentions that using npm as the key does not update pnpm-lock file at the moment. I think we should wait for a release to be made before merging this.

Reply:
Yeah good idea. So I close pr or put in draft ?

Reply:
Let's keep it as draft. Thank you for your contribution.

Reply:
You welcome 😁

Reply:
https://github.blog/changelog/2023-06-12-dependabot-version-updates-now-supports-pnpm/